Blockzilla/en.lproj/InfoPlist.strings

```diff
@@ -13,3 +13,5 @@
 /* (No Comment) */
 "NSPhotoLibraryUsageDescription" = "This lets you save and upload photos.";
 
+/* (No Comment) */
+"NSFaceIDUsageDescription" = "This lets you unlock the app.";
```

Comment:
Question: Do we need to do anything similar for TouchID? I didn't find anything in the 2 seconds I spent looking for it, but just want to make sure :)

Reply:
No similar key exists for Touch ID, so we do not need one. I confirmed this by checking the [full list of cocoa keys](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html)

Reply:
👍